controlsfx/src/main/java/impl/org/controlsfx/ImplUtils.java

```diff
@@ -157,4 +157,22 @@ public static void stripRootPane(Scene scene, Parent originalParent, boolean use
 
         return children;
     }
+
+    public static void focusPreviousSibling(Node node) {
+        final int index = node.getParent().getChildrenUnmodifiable().indexOf(node);
+        if (index != 0) {
+            node.getParent().getChildrenUnmodifiable().get(index - 1).requestFocus();

```

Comment:
If `indexOf` returns `-1`, won't this line tries to retrieve a children at index `-2` ?

Reply:
May be its better to use `if (index > 0)` ?

Reply:
Updated PR with this change.